modules/user-provided-certificates-for-default-ingress.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="user-provided-certificates-for-default-ingress_{context}"]
+= User-provided certificates for default ingress
+
+== Purpose
+
+Applications are usually exposed at
+`<app_name>.apps.<cluster_name>.<base_domain>`. You might want clients to
```

Comment:
I just realized the phrase "base domain" is badly overloaded.  For example, in the install config, the base domain might be "example.com", in which case the default base domain in the cluster DNS config would be "mycluster.example.com".  Perhaps it would be better to refer to the "ingress domain", which in this example would be "apps.mycluster.example.com".  The ingress domain is described in `networking/ingress/configuring_ingress_operator.adoc#nw-installation-ingress-config-asset_{context}`.